### ISSUE TITLE
Gate Botania behind mid-game RotaryCraft and ChromatiCraft progression

### ISSUE THREAD

**Issue by Toksyuryel** [Haggle1996/RevolutionPack#574]
One thing I have noticed in my time playing this pack is that Botania offers enormously powerful tools to the player very early on, and this pack makes it extremely easy to dive into as the only gating that has been done is to change manasteel to require carbon steel. This does little to prevent Botania from dominating the entire mid-game of the pack- as soon as I have steel I can make an entire set of manasteel tools that will last me from early game until the beginning of the endgame. To say nothing of all the other tools it provides. In short, I feel like Botania has become a comfort zone mod on the scale of how tcon used to be and in keeping with the pack's goals should be pushed further back in the progression to encourage players to branch out more.

To that end, my proposed changes are simple:
* Replace the gold in the mana spreader recipe with inductive ingot
* Change manasteel to use aura-conducting ingots

Both of these materials are solid mid-game for their respective mods and provide encouragement to actually progress in both and a suitably powerful reward for doing so (Botania tools are more powerful than what you have access to in either mod by that stage). And hopefully it will restore some semblance of tool progression to the pack.

**Comment by happy2pester** [comment]
As myself and Toksyuryel have already hashed out over the pack's IRC channel, I don't like this idea. I think it unnecessarily closes off some incredibly valuable earlygame tools, and makes the initial pack progression a lot harder. 

Tools are really resource intensive in this pack, especially in the really earlygame. At the cost of some pretty far ranging work, (for preference, munchdew, supported by endoflames) with botania, i can be secure in my basic tools and be in a position to access the rest of the content the pack has to offer. 

I'm personally in a position where i'm learning chromaticraft for the first time - and i've pushed hard on it to get at a HSLA hammer. I'm starting to figure out rotarycraft to move into higher tier ore refining. I'm working with techguns to gear up to progress in CrC further. 

My point is this: manasteel tools provide me with a stable platform on which to progress and actually enjoy this pack. From my point of view, especially without the questlines to push progression in other aspects (and yes, i know they're being worked on) i don't know enough about some of  the mods in this pack to actually make things like the inductive ingot and the aura conducting ingots. I'm learning! I'm currently the furthest I've ever been in CrC due to the push to get at the smeltery for my hammer. But i wouldn't have gotten as far as i have without the foundation of my manasteel tools.

To summarize: While i get that the point of this pack is to push people out of their comfort zones, i feel this change would be necessarily damaging to earlygame progression.

**Comment by Kataiser9** [comment]
I tend to agree with happy2pester, but there are two main points I'd like to add.

One, changing manasteel to require aura conducting ingots wouldn't just be a gating change - it'd also make most of Botania _much_ more expensive, even compared to ChC stuff. None of the ChC alloy ingots are at all cheap, especially in a pack where auto-farming large quantities of experience is difficult. Couple this with the fact that you need a lot more manasteel for Botania than you do alloy ingots for ChromatiCraft, and the liquid chroma requirements would get absurd very quickly. If Botania does get gated behind ChC in some way, this is not how I would recommend doing it.

Two, if I were Haggle, there's one other reason I'd seriously hesitate in making changes like this: gating/progression tweaks are already massively frontloaded in this pack, IMO. Getting up and running with basic ore doubling and iron/steel tools is a serious and fairly slow endeavor, but as you proceed beyond that point, player power starts to ramp up really freakin' fast. (This isn't really a criticism of the pack; Reika's mods make this borderline unavoidable with their high power level and lack of tweakability beyond the early stages.) Point is, we already have a fairly grueling early-game whose difficulty curve levels out quite abruptly, and I don't think it's a good idea to intensify that more than necessary.

**Comment by Barhandar** [comment]
I agree with both happy2pester and Kataiser9. Additionally, I have points to add.
- Chromaticraft is _heavily_ exploration-based. Botania not only enables solid through-game tools (for which there is _no replacement_ convenience-wise because easily repairable TiC tools are off), but it has the Sashes and the Ring of Chordata, facilitating early-game ability to explore. Forcing Chroma+Rotary progress to advance in Botania means that **you are cutting off both Chromaticraft and Botania, as well as forcing players to spend days upon days of dull walking before they can get pack's ores**.  
- The oceans are iced over due to bugs in Harder Wildlife and RTG, so there is no boat travel. Additionally, [Better Boat](https://mods.curse.com/mc-mods/minecraft/230898-better-boat) is not present in the pack, so even if the oceans weren't letting it go it's hell to travel over them without flight/waterbreathing/Thaumcraft progression, which are mid-game. So you're also forcing Chromaticraft to be focused on, because players _will_ have trouble getting fragments early on, and then they will need to go around collecting them once they get a jetpack before they can do Botania. By that point, majority of Botania's utilities will become _useless_ - and players will also hit Chroma's anti-focusing.
- There isn't an alternative to "branch out" to. There are no other tools in the pack that can be easily repaired, and irreparable tools are a massive inconvenience - especially since Silk Touch is required and Fortune is greatly desirable. All the alternatives to Botania's utilities are mid-to-late-game as well.
- Modifying Botania's progression to limit it isn't as easy as "stick hard-to-get items in recipes of constantly required stuff!!" and requires actually understanding said progression and how it interacts with the rest of the pack.

P.S.  

> Botania tools are more powerful than what you have access to in either mod by that stage

ChromaTiC tools are at the same progression stage, and Chroma Enchanter is significantly earlier, rendering advantages of Botania's tools other than "repair themselves" null.

**Comment by Kannei** [comment]
If any gating to botania was to be done without making Botania a mod noeone whould end up using (AS i see it whould be with Tok's sugestions), A simple change of making mana spreaders need a chroma alloy insted of gold to make the spreaders whould be enough. That whoul make u need to progres in chromaticraft enough to make the basic Chromaticraft alloy before starting botania. (appart from being abel to make some basic flowers and living rock+wood)

Seing as its a magic mod , needing something from RoC does not make as much sense.

**Comment by Gamerthekid** [comment]
Good points guys
